modules/quadbin/bigquery/doc/QUADBIN_SIBLING.md

```diff
@@ -0,0 +1,25 @@
+### QUADBIN_SIBLING
+
+{{% bannerNote type="code" %}}
+carto.QUADBIN_SIBLING(quadbin, direction)
+{{%/ bannerNote %}}
+
+**Description**
```

Comment:
Add any comment explaining that given the 0 resolution quadbin, the function shall return `null`